modules/dxp/apps/segments/segments-experiment-test/src/testIntegration/java/com/liferay/segments/experiment/web/internal/processor/test/SegmentsExperimentSegmentsExperienceRequestProcessorTest.java

```diff
@@ -633,6 +633,17 @@ public String getString(long companyId, String name) {
 			return _prefsProps.getString(companyId, name);
 		}
 
+		@Override
+		public String[] getStringArray(
+			long companyId, String name, String delimiter) {
+
+			if (Objects.equals("liferayAnalyticsEnableAllGroupIds", name)) {
+				return new String[] {"test"};
```

Comment:
liferayAnalyticsEnableAllGroupIds should return a boolean value: https://github.com/liferay/liferay-portal/blob/7de98649b260a6ed22d41ef22a25fce37de74298/modules/apps/analytics/analytics-settings-api/src/main/java/com/liferay/analytics/settings/configuration/AnalyticsConfiguration.java#L49

Reply:
done, thank you